README.md

```diff
@@ -107,6 +109,20 @@ Now you're ready to rumble!
 
 > Please note that this boilerplate is **production-ready and not meant for beginners**! If you're just starting out with react or redux, please refer to https://github.com/petehunt/react-howto instead. If you want a solid, battle-tested base to build your next product upon and have some experience with react, this is the perfect start for you.
 
+## When in doubt, re-install!
+If you're facing any inexplicable problems while installing dependencies, building your project or running tests, try reinstalling dependencies. It works for most cases. Run following commands in the exact order-
+
+Remove node_modules and build
+- `rm -rf node_modules`
+- `rm -rf build`
+
+Using yarn (recommended)
+- `yarn install`
+
+Using npm
+- `npm cache clean`
+- `npm install`
+
```

Comment:
I'm think this should go in Gatchas (don't make it look like trouble is expected). It would be good to reference gotchas from README, perhaps under the title of "Troubleshooting".